src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -86,6 +86,27 @@ def extract_images(k8s_obj: HikaruDocumentBase) -> Optional[Dict[str, str]]:
     return None
 
 
+def extract_image_list(k8s_obj: HikaruDocumentBase) -> List[str]:
+    containers_paths = [
+        [
+            "spec",
+            "template",
+            "spec",
+            "containers",
+        ],  # deployment, replica set, daemon set, stateful set, job
+        ["spec", "containers"],  # pod
+    ]
+    images = []
+    for path in containers_paths:
+        try:
+            for container in k8s_obj.object_at_path(path):
+                images.append(container.image)
+        except Exception:  # Path not found on object, not a real error
```

Comment:
Can you catch the exact error instead of all exception?

Reply:
There are many type of errors that can be thrown here.
I don't want to fail because I failed to extract the image